packages/TorneloScoresheet/src/pages/GraphicalRecording/MoveOptionsSheet.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Sheet from '../../components/Sheet/Sheet';
+import { colours } from '../../style/colour';
+import { ICON_CLOCK, ICON_HALF } from '../../style/images';
+import { PlayerColour } from '../../types/ChessGameInfo';
+import MoveOption from './MoveOption';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import { useGraphicalRecordingState } from '../../context/AppModeStateContext';
+
+export type EditingMove = {
+  colour: PlayerColour;
+  moveIndex: number;
+};
+
+type MoveOptionsSheetProps = {
+  editingMove: EditingMove | undefined;
+  dismiss: () => void;
+};
+
+const MoveOptionsSheet = ({ editingMove, dismiss }: MoveOptionsSheetProps) => {
+  const recordingState = useGraphicalRecordingState();
+  const actions = recordingState?.[1];
+
+  const toggleDraw = actions?.toggleDraw;
+
+  const handleDrawOffer = () => {
+    console.log(editingMove);
+    if (!toggleDraw || !editingMove) return;
+    toggleDraw(
+      editingMove.moveIndex * 2 +
+        (editingMove.colour === PlayerColour.Black ? 1 : 0),
+    );
+    dismiss();
+  };
+
+  return (
+    <Sheet
+      visible={Boolean(editingMove)}
+      dismiss={dismiss}
+      title="Move options">
+      <MoveOption
+        onPress={() => undefined}
+        colour={colours.primary}
+        icon={<ICON_CLOCK color={colours.white} />}
+        label="Record time"
+      />
+      <MoveOption
+        onPress={handleDrawOffer}
+        colour={colours.primary}
+        icon={<ICON_HALF color={colours.white} />}
+        label="Draw offer"
+      />
+      <MoveOption
+        onPress={() => undefined}
+        colour={colours.primary}
+        icon={<Icon name="edit" size={35} color={colours.white} />}
+        label="Edit move"
+      />
+      <MoveOption
+        onPress={dismiss}
+        colour={colours.secondary}
+        icon={<MaterialIcon name="cancel" size={35} color={colours.white} />}
+        label="Cancel"
```

Comment:
The cancel button isnt working for me? I dont understand why though

Reply:
Actually none of the buttons work for me. Even the console logging doesnt show up so i think the click is not being registered. Im on android

Reply:
Okay I think I've fixed this now. Let me know if it's still not working

Reply:
yeh all good working now :)